### PR TITLE
feat: Add kosko-charts autodevops action.

### PR DESCRIPTION
**Pull request by gary-van-woerkens**

### CODE REVIEW

**Comment by revolunet** [comment]
- needs rebase
 - .github/workflows/k8s-manifests-debug-test.yaml should not be here
 - README please

File: k8s-autodevops/action.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+name: "⛑ Generate manifests ⛑"
+description: "Generate k8s manifests via kosko-charts autodevops"
+inputs:
+  environment:
+    description: "The deployment envirnoment (dev | preprod | prod)"
```
Comment (revolunet) [inline k8s-autodevops/action.yml:5]:
```suggestion
    description: "The deployment environment (dev | preprod | prod)"
```

File: k8s-manifests-debug/action.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+name: "⛑ Debug manifests ⛑"
+description: "Debug some kubernetes manifests"
+inputs:
+  path:
+    description: "Your manifests YAML file path"
+  token:
+    description: "Github token for PR comment"
+  # env:
```
Comment (revolunet) [inline k8s-manifests-debug/action.yml:8]: `environment` also

**Review by douglasduteil — APPROVED** [review]
🎩

**Pull request merged**